### ISSUE TITLE
Add a deposit/withdraw tab 

### ISSUE THREAD

**Issue by jedmccaleb** [irisli/stellarterm#62]
Should add a few anchors like stellar desktop does to make it easy for people to get in and out.

**Comment by irisli** [comment]
Technical details:
- Make a new page on Accounts

**Comment by irisli** [comment]
Technical details:
- Make a new page on Accounts 

## UX Notes

### Currency first send
I realized that I have the send flow backwards. When someone wants to send money, they already know what currency they want to send. Additionally, in the Stellar network, they may want to send an asset to someone that has not trusted the asset!

The current flow asks the user for the recipient and only shows some assets that they can send. Unfortunately, it is confusing for someone who wants to send and asset but doesn't know that the recipient cannot receive it.

### "Send" and "Receive"
The UX on the Coinbase multi currency wallet is pretty well thought through (natural for a company with many employees). Basically, you can press `Send` to send elsewhere and `Receive` to get instructions on how to receive money. The emphasis for Stellar is on the *instructions* part. Many people using Stellar will be new to this concept and we must explain it nicely.

### Withdraw
There is a subtle distinction between sending and withdrawing. Sending could be thought of as a superset of withdrawing as you can send to others or also yourself. However, having `Send` and `Withdraw` coexist would be confusing.

We can make this distinction by having Withdraw be a smaller part of sending. And for some currencies, there can be special sending options such as `Send to BTC Address` as opposed to `Send to Stellar user`.

### UI aesthetics
For the purposes of this issue, don't worry about how beautiful the UI looks. The current UI for StellarTerm remains the 3 day hackjob. But in December/January, I will be overhauling everything and the UI will change al ot.

What is most important for this issue is getting done the backend and the content.

**Comment by jedmccaleb** [comment]
Anchors to add:
naobtc.com
apay.io
ripplefox.com
vcbear.net

**Comment by nikhilsaraf** [comment]
fyi, @irisli I'm working on this so feel free to assign to me if you want.

**Comment by irisli** [comment]
@nikhilsaraf Oh cool. GitHub won't let me assign external people but thats what the `work-in-progress-external` tag is for. Whether its actually clear is another question 😅 . But I don't think there will be a conflict. Thanks!

**Comment by irisli** [comment]
Also, feel free to ask me if you have questions.

**Comment by irisli** [comment]
And as with the way the StellarTerm css and js works, don't try to inherit existing structures and fork from it (such as special css classes or special cases in send). Simply just do a clone of them (just duplicate the css under a different name and the send ticker component). This isolates features so that taking down one feature or editing it won't break other things

**Comment by nikhilsaraf** [comment]
ok, thanks, will keep that in mind. I'm using this branch to make the commits, will have something for you to look at (PR) by tomorrow.

**Comment by irisli** [comment]
@nikhilsaraf Hey, lets try to avoid using react-select if we can. Right now, StellarTerm really only has two external depenencies: React and StellarSdk. Best to keep it low

**Comment by irisli** [comment]
But hmm, on other thought, if React Select makes things better, then it's fine to have it

**Comment by nikhilsaraf** [comment]
yes, that's what I'll be adding -- logos for currencies in the deposits dropdown, and we can use it in other places too to improve the dropdown selection process; it also has autocomplete so you can type to find anchors etc.

**Comment by irisli** [comment]
Oooh, maybe you can make it have two sizes.
```html
<AssetCard2Dropdown compact={true}>
<AssetCard2Dropdown compact={false}>
```

**Comment by nikhilsaraf** [comment]
https://github.com/irisli/stellarterm/pull/86 finished the first of 3 parts. Tracking all of them here.

- [x] Currency dropdown and anchor list in a new Deposits tab
- [ ] Add deposit instructions for each anchor
- [ ] Add icons for currency dropdown (optional UI/UX improvements)

**Comment by nikhilsaraf** [comment]
Added another PR/diff for the remaining 2 changes. It's based off the first PR (my repo) so I will need to re-submit the second PR once it's approved. here's the link: https://github.com/nikhilsaraf/stellarterm/pull/1
